jscomp/syntax/tests/printer/expr/expected/callback.res.txt

```diff
@@ -215,13 +215,16 @@ let f = () => {
   </div>
 }
 
-myPromise->Js.Promise.then_(value => {
+myPromise
+->Js.Promise.then_(value => {
```

Comment:
This is perhaps a bit unfortunate. Not so bad with multiple pipes, but does it also break with just one pipe if the function call is over multiple lines? E.g.:

```rescript
myPromise->Promise.then(v => {
  Js.log(v)
  Promise.resolve(v)
})


// vs

myPromise
->Promise.then(v => {
  Js.log(v)
  Promise.resolve(v)
})
```

Reply:
I will try to fix this edge case. Currently the latter will be formatted to the former. Thank you for the fast feedback.

Reply:
Yes, the former is what I'd prefer. It would be good to have a test for this, unless there already is one and I missed it.

Reply:
Just added your example to the tests.

Reply:
Awesome. Thank you!